network/src/main/scala/com/linkedin/norbert/network/netty/ClientChannelHandler.scala

```diff
@@ -212,7 +214,10 @@ class ClientStatisticsRequestStrategy(val stats: CachedNetworkStatistics[Node, U
       val nodeP = p.get(n).getOrElse(StatsEntry(0.0, 0, 0))
 
       val nodeMedian = doCalculation(Map(0 -> nodeP),Map(0 -> nodeN))
-      val available = nodeMedian <= clusterMedian * outlierMultiplier + outlierConstant
+      var available = true
```

Comment:
Modify the code to: `val nodeP = p.get(n).getOrElse(StatsEntry(0.0, 0, 0))

      val nodeMedian = doCalculation(Map(0 -> nodeP),Map(0 -> nodeN))

      val available = if (enableReroutingStrategies) {
        nodeMedian <= clusterMedian * outlierMultiplier + outlierConstant
      } else {
        !enableReroutingStrategies
      }`

Leave nodeMedian out of the  `if` loop because the scope issue since the callback function outside the `if` should use it. 
Change the logic a little: inside `else`, using `!enableReroutingStrategies` instead of `enableReroutingStrategies` because we want `available` to be true whenever `enableReroutingStrategies equal to `false`